helm/Chart.yaml

```diff
@@ -15,12 +15,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 4.0.2
+version: 4.1.0
```

Comment:
Not sure about this one @fatih-acar do you have a point of view ?

Reply:
We only update the patch version when the infrahub version patch is bumped, so you can go with 4.0.3 I guess?

Reply:
It was done with the invoke command, so gonna do it manually

Reply:
Ok, I think we can leave it like that then, it's mostly cosmetic to me